### PR TITLE
Implement vote modal and deposit modal

### PR DESCRIPTION
**Pull request by hochiw**
- Implement sign Vote and Deposit message api
- Implement VoteProposalForm, DepositProposalForm
- Implement VoteProposalModal, DepositProposalForm
- Misc fix for balance apis and input fileds

## Screenshots
Screenshot 2022-06-23 at 5 31 11 PM
Screenshot 2022-06-23 at 5 31 48 PM

refs oursky/likedao#121  oursky/likedao#122

### CODE REVIEW

File: react-app/src/api/bankAPI.ts [hunk 1]
```diff
@@ -31,7 +31,7 @@ export const useBankAPI = (): IBankAPI => {
       const balance = await cosmos.getBalance();
       const coinBalance = convertTokenToMinimalToken(balance.amount);
 
-      if (coinBalance.isLessThan(coinAmount)) {
+      if (coinBalance.isLessThanOrEqualTo(coinAmount)) {
```
Comment (chihimng) [inline react-app/src/api/bankAPI.ts:34]: but how about gas fee...
& allow use up all funds should be throw at balance < amount not <= amount? now it does not allow txn for that condition?

File: react-app/src/components/forms/DepositProposalForm/DepositProposalForm.tsx [hunk 2]
```diff
@@ -58,6 +58,7 @@ const DepositProposalForm: React.FC<DepositProposalFormProps> = (props) => {
         currencyUnit={coinDenom}
         inputProps={{
           placeholder: "0",
+          max: availableTokens.toNumber(),
```
Comment (chihimng) [inline react-app/src/components/forms/DepositProposalForm/DepositProposalForm.tsx:61]: will this cause issue for whale accounts?

Comment (chihimng) [reply react-app/src/components/forms/DepositProposalForm/DepositProposalForm.tsx:61]: assuming this is big number -> number down cast

**Review by elise-ng — CHANGES_REQUESTED** [review]

File: react-app/src/api/bankAPI.ts (hunk 1 above)
Comment (hochiw) [reply react-app/src/api/bankAPI.ts:34]: oh you right

**Comment by hochiw** [comment]
@chihimng updated 🙏

**Review by elise-ng — APPROVED** [review]

**Pull request merged**